BetterMD/elements/table.py

```
@@ -1,113 +1,487 @@
+    def from_pandas(cls, series:'pd.Series | pd.Index', head:'bool'=False):
+        try:
+            self = cls()
+
+            if head:
+                self.extend_children([Th(inner=[Text(d)]) for d in series])
+
+            self.extend_children([Td(inner=[Text(d)]) for d in series])
+
+            return self
+        except ImportError:
+            raise ImportError("`tables` extra is required to use `from_pandas`")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)